lib/ansible/modules/find.py

```diff
@@ -447,6 +476,22 @@ def statinfo(st):
     }
 
 
+def owner_filter(st, users, groups):
+    '''filter files that do not belong to a certain owner/group'''
+    if (users is None or (iter(users) and len(users) < 1)) and \
+       (groups is None or (iter(groups) and len(groups) < 1)):
+        return True
+    if (iter(users) and len(users) > 0) or (iter(groups) and len(groups) > 0):
+        stinfo = statinfo(st)
+        if iter(users) and len(users) > 0:
+            if stinfo['pw_name'] not in users:
+                return False
+        if iter(groups) and len(groups) > 0:
+            if stinfo['gr_name'] not in groups:
+                return False
```

Comment:
Calling `statinfo` for every path more than once feels inefficient. Can we convert the list of users and groups to a list of ints in `main()` for quick comparison? Then this could be:

```suggestion
    if users and st.st_uid not in users:
        return False
    if groups and st.st_gid not in groups:
        return False
```